src/app/templates/app/submit_new_license_namespace.html

```diff
@@ -383,4 +397,11 @@
     }
 });
 </script>
+<script type="text/javascript" src="{% static 'js/editor/codemirror.js' %}"></script>
+<script type="text/javascript" src="{% static 'js/editor/script.js' %}"></script>
```

Comment:
Why do we need all these scripts? It will increase load time.

Reply:
You are right. I did this during testing. I will remove them. Expect an update soon.